service_catalog/urls.py

```diff
@@ -40,14 +52,11 @@
     path('settings/catalog/service/<int:service_id>/operations/add/', views.add_service_operation,
          name='add_service_operation'),
     path('settings/catalog/service/<int:service_id>/operations/<int:operation_id>/delete/',
-         views.delete_service_operation,
-         name='delete_service_operation'),
```

Comment:
delete_service_operation or service_operation_delete?

Reply:
This belong to a previous PR. I'll update all URLs in a dedicated refactoring PR